tests/test_news_flash.py

```diff
@@ -232,6 +232,25 @@ def test_extract_location_text():
         assert expected_location_text == actual_location_text
 
 
+def test_location_extraction_geocode_extract():
+    if not secrets.exists("GOOGLE_MAPS_KEY"):
+        pytest.skip("Could not find GOOGLE_MAPS_KEY")
+
+    # this sentence would return nothing without try_rectify_location_string()
+    expected_location = {
+        'street': None,
+        'road_no': None,
+        'intersection': None,
+        'city': None,
+        'address': 'Lower Galilee, Israel',
+        'subdistrict': 'Kinneret',
+        'district': 'North District',
+        'geom': {'lat': 32.7461638, 'lng': 35.5028379},
```

Comment:
Just a note - the exact numbers make it volatile. It probably doesn't merit a dedicated "close to" predicate, but it might be worth at least a consideration, or a comment.

Reply:
Very true. That's why we have `mock.ANY`.
Fixed here, but this probably holds for other tests which also state the exact coordinates in the expected response,